OpenRA.Mods.Common/Traits/BotModules/McvManagerBotModule.cs

```diff
@@ -44,6 +44,9 @@ public class McvManagerBotModuleInfo : ConditionalTraitInfo
 		[Desc("Should deployment of additional MCVs be restricted to MaxBaseRadius if explicit deploy locations are missing or occupied?")]
 		public readonly bool RestrictMCVDeploymentFallbackToBase = true;
 
+		[Desc("Build additional MCV if cash is above this.")]
+		public readonly int BuildAdditionalMCVCashAmount = 1000;
```

Comment:
this seems a bit draconian no? Because often you need to expand in order to gain cash. It's not just a luxury expense

Reply:
changed to 2000